indico/modules/categories/blueprint.py

```diff
@@ -16,9 +16,12 @@
 
 from __future__ import unicode_literals
 
-from indico.modules.categories.controllers import RHCategoryStatistics
+from indico.modules.categories.controllers import (RHCategoryStatistics, RHCategoryMoveContents,
+                                                   RHCategoryInfo)
 from indico.web.flask.wrappers import IndicoBlueprint
 
 _bp = IndicoBlueprint('categories', __name__, template_folder='templates', url_prefix='/category')
 
 _bp.add_url_rule('/<categId>/statistics', 'category-statistics', RHCategoryStatistics)
+_bp.add_url_rule('/<categId>/manage/move/', 'move-contents', RHCategoryMoveContents)
+_bp.add_url_rule('/<categId>/info', 'info', RHCategoryInfo, methods=('POST',))
```

Comment:
ah I see, the change to the locator is not in upstream/devel yet. adding a new commit for it (it's currently only in d1a0f092bb53d0b45a5314339f94063a80b85a46 but that's huge and WIP)